app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -81,6 +91,17 @@ public void bind(@NonNull final CompositeDisposable d) {
                           .observeOn(computation())
                           .subscribe(searchViewModel::search,
                                      e -> e(e, "Error getting changed text")));
+
+            d.add(searchViewModel.getSearchErrorOnceAndStream()
+                                 .observeOn(mainThread())
+                                 .subscribe(throwableOption -> {
+                                                if (throwableOption.isSome()) {
```

Comment:
Make the String "There was an error searching" a String resource. Retrieve using `ResourceProvider`.